core/examples/luigi-example-js/webpack.config.js

```diff
@@ -1,24 +1,35 @@
 const CopyPlugin = require('copy-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
 
 module.exports = {
```

Comment:
When I build the whole project everything will copied to the created dist folder and the resources won't copied to our public folder. That means that the newest luigi will not be served.
If we specify the _output_ target it works.
```suggestion
const path = require('path');

module.exports = {
  output: {
    path: path.resolve(__dirname, 'public'),
  },
  plugins: [

```